### ISSUE TITLE
Collapse more categories with a ton of options

### ISSUE THREAD

**Issue by Ennazus** [psrc/travel-study-stories#16]

**Comment by cyyllam** [comment]
Yes, I tested Travel:Detailed Mode in the Simple Table and the labels on the plot were very squished

**Comment by cyyllam** [comment]
If we are creating collapsed categories (i.e new categories) we will also need to add to variables_values.xlsx and 'rank' the order of those values for sorting purposes. 

Currently noticing some variables like 'Age group' where the values are not sorted in a logical way

**Comment by Ennazus** [comment]
Use the HHSurveyDataExplorerValues2017 field ValueGroup1 to collapse categories
